### ISSUE TITLE
Incorrect implicitly label for partial files

### ISSUE THREAD

**Issue by mrmlnc** [mrmlnc/vscode-scss#99]
- VS Code Version: 1.40.2
- SCSS IntelliSense Version: 0.8.1+
- Operating System: Windows

Reproducible Case:

```scss
// _provider.scss
@mixin gradient-striped($color: rgba(255, 255, 255, .15), $angle: 45deg) {
  background-image: -webkit-linear-gradient($angle, $color 25%, transparent 25%, transparent 50%, $color 50%, $color 75%, transparent 75%, transparent);
  background-image: -o-linear-gradient($angle, $color 25%, transparent 25%, transparent 50%, $color 50%, $color 75%, transparent 75%, transparent);
  background-image: linear-gradient($angle, $color 25%, transparent 25%, transparent 50%, $color 50%, $color 75%, transparent 75%, transparent);
}

.test1 {
  @include gradient-striped;
  background-size: 40px 40px;
}

// test.scss
@use "provider" as p;

.test2 {
  @include p.gradient-striped;
  background-size: 40px 40px;
}
```

Steps to Reproduce:

1. Open the `test.scss` file;
2. Hover the `gradient-striped` node; The lettering would be `@import "_provider.scss";`; Go to the definition;
3. Save the `_provider.scss` file;
4. Back to the `test.scss` file
5. Hover the `gradient-striped` node; The lettering would be `@import "_provider.scss"; (implicitly)`;

**Comment by mrmlnc** [comment]
We need to use `ls.findDocumentLinks2` to fix this. To do this, we must switch to the URI instead of the paths in the providers, parser and scanner.

https://github.com/microsoft/vscode-css-languageservice/blob/2a87ef7ac2ce9eb4db3583e38a98bbac2c6464bf/src/services/scssNavigation.ts#L48

**Comment by mrmlnc** [comment]
Fixed by `0.9.0`.

**Comment by mrmlnc** [comment]
Must be fixed by `0.9.0`.
